src/iam_roles/iamrolesservicemapping.py

```diff
@@ -55,7 +78,8 @@ def lambda_handler(event, context):
         role_arn = resource_file_content[iterator]["Arn"]
         role_name = resource_file_content[iterator]["RoleName"]
         if len(resource_file_content[iterator]["RoleLastUsed"]) != 0:
-            role_region = resource_file_content[iterator]["RoleLastUsed"]["Region"]
+            region_id = resource_file_content[iterator]["RoleLastUsed"]["Region"]
```

Comment:
I specifically can't specify on exact line but here region_id should be passed this boto3 api will not work with region name  on line # 117
                             service_client = boto3.client(
                                resource, region_name=role_region
                            )

Reply:
updated https://github.com/XgridInc/xc3/commit/131ca5a9d982dca11cc2842e2da5fe3dcbf159eb